src/main/java/com/epam/bookstoreservice/model/IntegerModel.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.bookstoreservice.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * the model for Integer
+ */
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class IntegerModel extends RepresentationModel<IntegerModel> {
```

Comment:
You could move this model to a new package called 'hateoas/model' since this feature is related to it.